Multisig/Logic/AuthenticationController.swift

```diff
@@ -142,55 +141,158 @@ class AuthenticationController {
         }
     }
 
-    var isBiometryPossible: Bool {
-        guard let user = user else { return false }
-        return (try? accessService.isAuthenticationMethodPossible(userID: user.id, method: .biometry)) ?? false
```

Comment:
Do you plan to extend the library later?

Reply:
no, I even plan to stop using it as it is not needed anymore